### PR TITLE
dev-db/mysqltuner: bump to 1.7.15

### PR DESCRIPTION
**Pull request by hsntgm**
Version bump to 1.7.15. Also introduces new
patch that fixes password detecting error.

Package-Manager: Portage-2.3.66, Repoman-2.3.16
Signed-off-by: Hasan ÇALIŞIR <hasan.calisir@psauxit.com>

### CODE REVIEW

File: dev-db/mysqltuner/mysqltuner-1.7.15.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+HOMEPAGE="http://www.mysqltuner.com/"
+SRC_URI="https://github.com/major/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	virtual/mysql
+	dev-lang/perl
```
Comment (juippis) [inline dev-db/mysqltuner/mysqltuner-1.7.15.ebuild]: Please sort this list :)

File: dev-db/mysqltuner/mysqltuner-1.7.15.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	virtual/perl-Getopt-Long"
+
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	# Fix password detecting under MySQL 5.7 upstream #442 #427 #441
+	"${FILESDIR}/${P}-password.patch" )
```
Comment (juippis) [inline dev-db/mysqltuner/mysqltuner-1.7.15.ebuild]: Please also try getting this patch applied into upstream. It's not a showstopper now, but it's always nicer to drop carrying around patches forever.

File: dev-db/mysqltuner/mysqltuner-1.7.15.ebuild [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=MySQLTuner-perl
+
+DESCRIPTION="MySQLTuner is a high-performance MySQL tuning script"
+HOMEPAGE="http://www.mysqltuner.com/"
```
Comment (juippis) [inline dev-db/mysqltuner/mysqltuner-1.7.15.ebuild]: You could also add https://github.com/major/MySQLTuner-perl here.

File: dev-db/mysqltuner/mysqltuner-1.7.15.ebuild [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+SRC_URI="https://github.com/major/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```
Comment (juippis) [inline dev-db/mysqltuner/mysqltuner-1.7.15.ebuild]: It seems to be GPL-3+ ;) if you look at perl scripts in their Github.

File: dev-db/mysqltuner/mysqltuner-1.7.15.ebuild (hunk 2 above)
Comment (hsntgm) [reply dev-db/mysqltuner/mysqltuner-1.7.15.ebuild]: Upstream aware of that issue ( #442 #427 #441 ) but haven't been fixed yet. I implemented quick fix.

File: dev-db/mysqltuner/mysqltuner-1.7.15.ebuild (hunk 1 above)
Comment (hsntgm) [reply dev-db/mysqltuner/mysqltuner-1.7.15.ebuild]: Done.

File: dev-db/mysqltuner/mysqltuner-1.7.15.ebuild (hunk 3 above)
Comment (hsntgm) [reply dev-db/mysqltuner/mysqltuner-1.7.15.ebuild]: Done.

File: dev-db/mysqltuner/mysqltuner-1.7.15.ebuild (hunk 4 above)
Comment (hsntgm) [reply dev-db/mysqltuner/mysqltuner-1.7.15.ebuild]: Done.

**Comment by juippis** [comment]
Thanks!

**Pull request closed without merging**